packages/react/src/stories/ic-hero.stories.mdx

```diff
@@ -55,6 +55,81 @@ export const ControlledReset = () => {
   );
 };
 
+### Playground
+
+export const defaultArgs = {
+  aligned: "left",
+  backgroundImage: image,
+  contentAligned: "left",
+  disableBackgroundParallax: false,
+  heading: "Hero heading",
+  secondaryHeading: "Secondary Heading",
+  secondarySubheading: "This is a secondary description.",
+  size: "default",
+  subheading: "Hero description. This is a Hero component, it should be used as a page heading.",
+};
+
+<Canvas>
+  <Story
+    args={defaultArgs}
+    name="Playground"
+    parameters={{ layout: "fullscreen" }}
+    argTypes={{
+      aligned: {
+        options: ["center", "full-width", "left"],
+        control: { type: "select" },
+      },
+      backgroundImage: {
+        control: { type: "text" },
+      },
+      contentAligned: {
+        options: ["center", "left"],
+        control: { type: "select" },
+      },
+      disableBackgroundParallax: {
+        control: { type: "boolean" },
+      },
+      heading: {
+        control: { type: "text" },
+      },
```

Comment:
For any text ones I don't think you have to add a control type, it infers it from the default arg being a string. Same for boolean ones too